packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -948,10 +952,99 @@ QUnit.module('Chat', () => {
 
             this.clock.tick(timeout);
 
+            assert.deepEqual(this.instance.option('items'), [...messages, newMessage], 'items option should contain all messages including the new one');
+            assert.strictEqual(this.getBubbles().length, 3, 'new message should be rendered in list');
+        });
+
+        QUnit.test('new message should be rendered when using store.push({ type: insert }) and reloadOnChange is true', function(assert) {
```

Comment:
```suggestion
        QUnit.test('new message should be rendered after message is rendered if reloadOnChange is true', function(assert) {
```

Reply:
I'm not sure that new test name is informative, it looks like tautology

Reply:
after message is entered*

Reply:
Fixed